### DISCUSSION TITLE
What's the maximum size of logs can Airflow display normally on webserver UI?

### DISCUSSION THREAD

**Discussion by ORuteMa** [apache/airflow#28720] [Q&A] [answered]
I have a Spark operator task which generate over 50MB size log, and it can not be display on webserver ui normally. What's worse is that Airflow webUI will stuck once I open up the log page, I think Airflow UI should decide whether to display according to the log file size.

**Comment by potiuk** [comment, marked as answer]
Better solution will be to limit your logs to something meaningful but yeah i f you would like to implement a limit check - PR for that will be welcome.

**Comment by donbreo** [comment]
So.... was there a PR for this? :) 
I am using airflow to run a supplier call containing more than half a million hotels' data. Despite restricting the logs to something meaningful its still going to be very long.
